mint-auditor/src/http_api/service.rs

```diff
@@ -49,12 +49,41 @@ impl MintAuditorHttpService {
 
         Ok(BlockAuditDataResponse::new(block_audit_data, balances))
     }
+
+    /// Get a paginated list of audited mints, along with corresponding mint tx
+    /// and gnosis safe deposit
+    pub fn get_audited_mints(
+        &self,
+        offset: Option<u64>,
+        limit: Option<u64>,
+    ) -> Result<Vec<AuditedMintResponse>, Error> {
+        let conn = self.mint_auditor_db.get_conn()?;
+
+        let query_result = AuditedMint::list_with_mint_and_deposit(offset, limit, &conn)?;
+
+        let response = query_result
+            .iter()
```

Comment:
I believe that you can replace this with `.into_iter()` and then drop the clones below